packages/mobile/src/components/comments/CommentThread.tsx

```diff
@@ -62,7 +66,8 @@ export const CommentThread = (props: CommentThreadProps) => {
               {rootComment?.replies?.map((reply: ReplyComment) => (
                 <Flex w='100%' key={reply.id}>
                   <CommentBlock
-                    comment={reply}
+                    // TODO: Determine proper typing for comments and replies
```

Comment:
would a reply be a comment with a parent_comment_id? feels like Comment would be the right type in that case

Reply:
ah just saw the models above, nvm

Reply:
yeah what do you think is the move? @DejayJD for context Comment and ReplyComment have different types now since ReplyComment doesn't have replyCount, but a lot of our code expects ReplyComment and Comment to have the same signatures.

Reply:
I also ended up deviating the `Comment` and `ReplyComment` models on my branch as well. 
Every component that can do both replies & root comments I just unioned the types `Comment | ReplyComment` and type narrowed when needing the non-shared things.

e.g. https://github.com/AudiusProject/audius-protocol/pull/9670/files#diff-bd0d6269d59a3a176a3d299a8f02ab1a0b61d401a76c6f451f70b29cd3a3fa62R269

Reply:
Probably you can wait for my PR to merge which I think should set up the unions for you and just need to tweak where you need `replyCount` probably

Reply:
Sounds like a plan!